onnx/backend/test/case/node/abs.py

```diff
@@ -10,17 +10,77 @@
 from . import expect
 
 
+def generate_test(x, name):
```

Comment:
annotations here, describe the data type of `x` and `name`, and also return value of this function.